<launch>
    <arg name="robot_description_path" />
    <arg name="namespace_prefix" default="robot_namespace" />
    
    <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen" ns="$(arg namespace_prefix)" >
        <rosparam file="$(find project_turtleplan)/config/costmap_common_params.yaml" command="load" ns="global_costmap" />
        <rosparam file="$(find project_turtleplan)/config/costmap_common_params.yaml" command="load" ns="local_costmap" />
        <rosparam file="$(find project_turtleplan)/config/local_costmap_params.yaml" command="load" />
        <rosparam file="$(find project_turtleplan)/config/global_costmap_params.yaml" command="load" />
        <rosparam file="$(find project_turtleplan)/config/base_local_planner_params.yaml" command="load" />
    </node>
</launch>
